config/config.yml_example

```diff
@@ -192,8 +192,10 @@ oauth:
   client_id: 
   client_secret: 
   callback_urls:
-    - http://vouch.yourdomain.com:9090/auth
-    - http://vouch.yourotherdomain.com:9090/auth
+    - url: http://vouch.yourdomain.com:9090/auth
+      domain: yourdomain.com
```

Comment:
I don't think this should require a change to the configuration.  I don't see a reason why `domains` couldn't be matched to `callback_urls` at startup.

- if using multiple callback_urls
  - sort all `domains` by length descending (I think the code already does this)
  - same for the domain portion of each of the `callback_urls`
  - walk each callback_url.domain and test for first match from `domains` (test from the right)
  - map callback_url to domain (for later use in oauthLoginURL)
  - warn/error/fail if something doesn't align properly